build/azure-pipelines/cli/test.yml

```diff
@@ -0,0 +1,26 @@
+parameters:
+  - name: VSCODE_CLI_TARGETS
+    default: []
+    type: object
+  - name: VSCODE_CLI_DIR
+    type: string
+    default: './'
+  - name: VSCODE_CLI_BINARY_NAME
+    type: string
+  - name: VSCODE_CLI_RUST_CHANNEL
+    type: string
+    default: stable
+
```

Comment:
Should this be configurable?

Reply:
I don't think so. We will never want to publish a build (certainly not from main at least) that's on a non-stable Rust version.